src/api_server.cc

```diff
@@ -139,4 +147,28 @@ void MetadataApiServer::HandleMonitoredResource(
   }
 }
 
+void MetadataApiServer::HandleHealthz(
+    const HttpServer::request& request,
+    std::shared_ptr<HttpServer::connection> conn) {
+  if (health_checker_->IsHealthy()) {
```

Comment:
Should we allow `health_checker_` to be `nullptr`, so we can test different handlers in isolation?

Reply:
Done.